Docs/source/highlights.rst

```diff
@@ -154,6 +154,11 @@ High Energy Astrophysical Plasma Physics
 
 Scientific works in astrophysical plasma modeling.
 
+#. Ghosh S, Bhat P.
+   **Magnetic Reconnection: An Alternative Explanation of Radio Emission in Galaxy Clusters**.
+   The Astrophysical Journal **979** 1, 2025.
```

Comment:
Thanks for adding this! I think the full name of the journal is The Astrophysical Journal Letters:
```suggestion
   The Astrophysical Journal Letters **979** 1, 2025.
```

Reply:
True, maybe we should fix the entry below, too.

Reply:
Ah no, they are indeed different.